src/custom-types.json

```diff
@@ -78,7 +78,11 @@
   },
   "application/javascript": {
     "charset": "UTF-8",
-    "compressible": true
+    "compressible": true,
+    "extensions": ["mjs"],
+    "sources": [
+      "https://github.com/nodejs/node-eps/blob/master/002-es-modules.md"
```

Comment:
https://github.com/nodejs/node-eps/pull/61 is approved pending confirmation of what the preferred MIME type should be. Either `text/javascript` or `application/javascript`.